pennylane_lightning/core/src/lightning.qubit.toml

```diff
@@ -0,0 +1,79 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+version = "v0.31.0"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+[[operations.gates]]
+full = [
+	"Identity",
+	"PauliX",
+	"PauliY",
+	"PauliZ",
+	"Hadamard",
+	"S",
+	"T",
+	"PhaseShift",
+	"RX",
+	"RY",
+	"RZ",
+	"Rot",
+	"CNOT",
+	"CY",
+	"CZ",
+	"SWAP",
+	"IsingXX",
+	"IsingXY",
+	"IsingYY",
+	"IsingZZ",
+	"ControlledPhaseShift",
+	"CRX",
+	"CRY",
+	"CRZ",
+	"CRot",
+	"CSWAP",
+	"Toffoli",
+	"MultiRZ",
+]
+# QubitUnitary should never be here.
+# Gates in "matrix" are decomposed to QubitUnitary.
+# Having a QubitUnitary here would mean an infinite loop.
+matrix = [
+	"OrbitalRotation",
+	"MultiControlledX",
```

Comment:
I added an issue to keep track of this: https://github.com/PennyLaneAI/catalyst/issues/380 . I will not be implementing this one to avoid spending too much time on it, but it should be relatively straight forward. I will however work on adding the Single/Double excitations to the runtime.